src/index.ts

```diff
@@ -40,7 +40,7 @@ export {default as Overlay, OverlayTypes} from './components/overlay';
 export {default as RadioButton, RadioButtonPropTypes, RadioButtonProps} from './components/radioButton/RadioButton';
 export {default as RadioGroup, RadioGroupPropTypes, RadioGroupProps} from './components/radioButton/RadioGroup';
 export {default as SectionsWheelPicker, SectionsWheelPickerProps} from './components/sectionsWheelPicker';
-export {default as SegmentedControl, SegmentedControlProps} from './components/segmentedControl';
+export {default as SegmentedControl, SegmentedControlProps, SegmentedControlItemProps} from './components/segmentedControl';
 export {default as Switch, SwitchProps} from './components/switch';
```

Comment:
I don't think this one is part of the PR (: 
Merging master will probably fix that

Reply:
There was a conflict here after merging master